hugo/content/contact.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Contact
+toc: true
+---
+
+# Contact
+
```

Comment:
This adds a duplicate headline to the rendered page. Did you miss that on testing?

```suggestion

```

Reply:
Seems that way. Ok, will use the title tag only instead.